<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GDBackup</string>
  </property>
  <widget class="QWidget" name="miCentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="tableBackups">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>731</width>
         <height>271</height>
        </rect>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>152</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Folder name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Creation date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Assets</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>70</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="assetsCheckbox">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>380</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 12pt</string>
       </property>
       <property name="text">
        <string>Assets (songs, sfx)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectFolder">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>340</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backupData">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>340</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Backup</string>
       </property>
      </widget>
      <widget class="QLabel" name="backupLocation">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>531</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14pt;
</string>
       </property>
       <property name="text">
        <string>Backups</string>
       </property>
      </widget>
      <zorder>tableBackups</zorder>
      <zorder>pushButton</zorder>
      <zorder>assetsCheckbox</zorder>
      <zorder>backupData</zorder>
      <zorder>backupLocation</zorder>
      <zorder>label</zorder>
      <zorder>selectFolder</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
